### PR TITLE
Feature/run generation from folders

### PR DESCRIPTION
**Pull request by mor-m-shemesh**
This is a template - 
there will need to be a complementing PR that implements GenerationManager.save_to_folder

### CODE REVIEW

File: src/corona_hakab_model/main.py [hunk 1]
```diff
@@ -14,30 +16,58 @@
 from supervisor import LambdaValueSupervisable, Supervisable, SimulationProgression
 
 
-
 from typing import TYPE_CHECKING
 if TYPE_CHECKING:
     import pandas as pd
 
+logger = logging.getLogger('application')
+logger.setLevel("Info")
```
Comment (RoyIsland) [inline src/corona_hakab_model/main.py:24]: switch from "Info" to logging.INFO ("Info" gives me error when i run it)

File: src/corona_hakab_model/main.py [hunk 2]
```diff
@@ -102,6 +132,56 @@ def main():
...
+    else:
+        circles_consts = CirclesConsts()
+
+    if args.matrix_consts_path:
+        matrix_consts = MatrixConsts.from_file(args.matrix_consts_path)
+    else:
+        matrix_consts = MatrixConsts()
+
+    gm = GenerationManger(circles_consts=circles_consts, matrix_consts=matrix_consts)
+    gm.save_to_folder(args.output_folder)
```
Comment (RoyIsland) [inline src/corona_hakab_model/main.py:157]: save_to_folder not exists

**Comment by RoyIsland** [comment]
`    if args.sub_command == "generate":
        with open(args.output, "wb") as w:
            write(gm.matrix_data.matrix, w)
        return`
this if will never be true

File: src/corona_hakab_model/main.py [hunk 3]
```diff
@@ -102,6 +132,56 @@ def main():
...
+        generate_from_folder(args.consts_folder)
+        return
+    elif args.master_folder:
+        generate_from_master_folder(args.master_folder)
+        return
+
+    if not args.output_folder:
+        logger.error("No output folder given! use --output-folder")
+
+    if args.circles_consts_path:
```
Comment (RoyIsland) [inline src/corona_hakab_model/main.py:146]: extract this part to function
`    if args.circles_consts_path:
        circles_consts = CirclesConsts.from_file(args.circles_consts_path)
    else:
        circles_consts = CirclesConsts()

    if args.matrix_consts_path:
        matrix_consts = MatrixConsts.from_file(args.matrix_consts_path)
    else:
        matrix_consts = MatrixConsts()`
Same code in 2 places

Comment (mor-m-shemesh) [reply src/corona_hakab_model/main.py]: Done

File: src/corona_hakab_model/main.py (hunk 1 above)
Comment (mor-m-shemesh) [reply src/corona_hakab_model/main.py]: Done, I fixed it but overrode my own changes in a rebase

**Pull request merged**